protocols/bgp/metrics/bgp_address_family_metrics.go

```diff
@@ -0,0 +1,17 @@
+package metrics
+
+// BGPAddressFamilyMetrics provides metrics on AFI/SAFI level for one session
+type BGPAddressFamilyMetrics struct {
```

Comment:
I miss a metric representing the state of the FSM. 
Currently I compute the following metric:
```
prometheus.NewDesc(
		"peer_states_total",
		"peer states of all connected peers",
		[]string{"peer", "state"},
		nil,
	)
```
However I can not access the required information with the current master since `server.PeerInfo` is gone.

Reply:
It now exists, without having duplicate states to manage. Thanks for noticing.